css/styles.css

```diff
@@ -0,0 +1,177 @@
+.rules-container {
+    width: 95vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.hidden {
+    display: none;
+}
+
+.rules-header {
+    display: flex;
+    justify-content: center;
+    align-self: center;
+}
+
+.rules-title {
+    margin-right: 1rem;
+}
+
+.rules-heading {
+    padding-left: 2.5rem;
+}
+
+ul {
+    font-size: 1.2rem;
+}
+
+* {
+    font-family: 'Bebas Neue', sans-serif;
+    font-weight: 800;
+}
+
+h1 {
+    text-align: center;
+}
+
+hr {
+    width: 80%;
+    border: 0.1rem solid #333333;
+}
+
+.game {
+    width: 90%;
```

Comment:
Doesn't look like this width actually does anything

Reply:
you are correct - I have removed